### PR TITLE
Codespell fixups

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

File: pkg/specgenutil/volumes.go [hunk 1]
```diff
@@ -272,7 +272,7 @@ func Mounts(mountFlag []string, configMounts []string) (map[string]spec.Mount, m
 	}
 
 	// If user specified a mount flag that conflicts with a containers.conf flag, then ignore
-	// the duplicate. This means that the parseing of containers.conf configMounts, should always
+	// the duplicate. This means that the parsing of containers.conf configMounts, should always
```
Comment (edsantiago) [inline pkg/specgenutil/volumes.go:275]: Unnecessary comma after `configMounts`

File: cmd/quadlet/main.go [hunk 2]
```diff
@@ -160,7 +160,7 @@ func appendSubPaths(dirs []string, path string, isUserFlag bool, filterPtr func(
 }
 
 func nonNumericFilter(_path string, isUserFlag bool) bool {
-	// when running in rootless, only recrusive walk directories that are non numeric
+	// when running in rootless, only recursive walk directories that are non numeric
```
Comment (edsantiago) [inline cmd/quadlet/main.go:163]: This entire comment is hard to parse. Maybe "...only recurse into directories..."? Or "...only walk directories...", with the assumption that `walk` implies recursion?

Also, `non-numeric` is a compound adjective and should be hyphenated. Or written as "NOT numeric."

Also, punctuation. If I'm reading the comment correctly, maybe it should read "...are non-numeric SO AS TO ignore ..."?

**Review by edsantiago — CHANGES_REQUESTED** [review]
Grammar problems in addition to spelling

**Review by edsantiago — APPROVED** [review]
LGTM

**Review by Luap99 — COMMENTED** [review]
/lgtm
